parlai/core/torch_generator_agent.py

```diff
@@ -1777,7 +1817,7 @@ def __init__(self, *args, **kwargs):
             raise ValueError('Greedy search can only be run with beam size 1.')
 
     def select_paths(self, logprobs, prior_scores, current_length) -> _PathSelection:
-        tok_scores, tok_ids = logprobs.max(1)
+        tok_scores, tok_ids = logprobs
```

Comment:
I don't know this code well. I don't understand why did you have to change this part? isn't this for greedy? And I don't find any indication that the general code was changed to justify changing this part.

Reply:
good catch, this is an artifact, will fix